src/picongpu/include/particles/pusher/particlePusherFree.hpp

```diff
@@ -42,7 +43,7 @@ namespace picongpu
             {
 
                 Velocity velocity;
-                const PosType vel = velocity(mom, mass);
+                const float3_X vel = velocity(mom, mass);
```

Comment:
Would that have any advantage? If so, I will change it.

I chose to use `float3_X` to keep the same structures as in the other pushers.